### ISSUE TITLE
Parsing error on french layout template

### ISSUE THREAD

**Issue by AlexGiroud** [jhipster/generator-jhipster#20411]
##### **Overview of the issue**

When creating an application with the generator and selecting Angular/AdminUI/No internationalization support/French a parsing error appears ; 

`Error parsing file src\main\webapp\app\layouts\error\error.route.ts: SyntaxError: ',' expected. (10:26)`

##### **Reproduce the error**

- May JHipster anonymously report usage statistics to improve the tool over time? No
- Which *type* of application would you like to create? Monolithic application (recommended for simple projects)
- What is the base name of your application? redacted
- Do you want to make it reactive with Spring WebFlux? No
- What is your default Java package name? com.redacted.redacted
- Which *type* of authentication would you like to use? OAuth 2.0 / OIDC Authentication (stateful, works with Keycloak
and Okta)
- Which *type* of database would you like to use? SQL (H2, PostgreSQL, MySQL, MariaDB, Oracle, MSSQL)
- Which *production* database would you like to use? PostgreSQL
- Which *development* database would you like to use? H2 with disk-based persistence
- Which cache do you want to use? (Spring cache abstraction) Ehcache (local cache, for a single node)
- Do you want to use Hibernate 2nd level cache? Yes
- Would you like to use Maven or Gradle for building the backend? Gradle
- Do you want to use the JHipster Registry to configure, monitor and scale your application? No
- Which other technologies would you like to use?
- Which *Framework* would you like to use for the client? **Angular**
- Do you want to generate the admin UI? **Yes**
- Would you like to use a Bootswatch theme (https://bootswatch.com/)? Default JHipster
- Would you like to enable internationalization support? **No**
- Please choose the native language of the application **French**
- Besides JUnit and Jest, which testing frameworks would you like to use?
- Would you like to install other generators from the JHipster Marketplace? No

##### **Suggest a Fix**

Escape the ' line 10  of  the angular file (webapp\app\layouts\error\error.route.ts) for the french version

I don't know if you need to escape it in the translation or in the function call, but the translation file seems to be there: generators/languages/templates/src/main/webapp/i18n/fr/error.json

##### **JHipster Version(s)**

v7.9.3

##### **Browsers and Operating System**

Tested on Windows 10 and fedora

- [x] Checking this box is mandatory (this is just to show you read everything)

**Comment by dwarakaprasad** [comment]
@AlexGiroud can you please run jhipster info and paste the output to the defect.

when do you get this error? code generation or application startup

**Comment by AlexGiroud** [comment]
I get this error on the first code generation.

Here is the output of jhipster info : 

Welcome to JHipster v7.9.3

Welcome to the JHipster Information Sub-Generator

##### **JHipster Version(s)**

```
testjhipster@0.0.1-SNAPSHOT C:\Users\redacted\Documents\testjhipster
`-- (empty)

```


##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**


<details>
<summary>.yo-rc.json file</summary>
<pre>
{
  "generator-jhipster": {
    "applicationType": "monolith",
    "authenticationType": "oauth2",
    "baseName": "testjhipster",
    "blueprints": [],
    "buildTool": "gradle",
    "cacheProvider": "ehcache",
    "clientFramework": "angularX",
    "clientPackageManager": "npm",
    "clientTheme": "none",
    "clientThemeVariant": "",
    "creationTimestamp": 1669082304299,
    "databaseType": "sql",
    "devDatabaseType": "h2Disk",
    "devServerPort": 4200,
    "dtoSuffix": "DTO",
    "enableGradleEnterprise": false,
    "enableHibernateCache": true,
    "enableSwaggerCodegen": false,
    "enableTranslation": false,
    "entitySuffix": "",
    "jhiPrefix": "jhi",
    "jhipsterVersion": "7.9.3",
    "languages": ["en", "fr"],
    "messageBroker": false,
    "microfrontend": false,
    "microfrontends": [],
    "nativeLanguage": "fr",
    "otherModules": [],
    "packageName": "com.mycompany.myapp",
    "pages": [],
    "prodDatabaseType": "postgresql",
    "reactive": false,
    "searchEngine": false,
    "serverPort": "8080",
    "serverSideOptions": [],
    "serviceDiscoveryType": "no",
    "skipCheckLengthOfIdentifier": false,
    "skipClient": false,
    "skipFakeData": false,
    "skipUserManagement": true,
    "testFrameworks": [],
    "websocket": false,
    "withAdminUi": true
  }
}

</pre>
</details>


##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

<details>
<summary>JDL entity definitions</summary>

<pre>

</pre>
</details>


##### **Environment and Tools**

openjdk version "18.0.2.1" 2022-08-18
OpenJDK Runtime Environment (build 18.0.2.1+1-1)
OpenJDK 64-Bit Server VM (build 18.0.2.1+1-1, mixed mode, sharing)

git version 2.24.0.windows.2

node: v18.12.1

npm: 8.19.2

**Comment by jminkhar** [comment]
Got the same error
![image](https://user-images.githubusercontent.com/48157922/204111664-55cb14a9-b001-4246-8723-b28c66e57f49.png)

**Comment by djibythiaw** [comment]
Hello,
I have the same error than @jminkhar with french language without the internationalization. 
Do you have a solution ?

**Comment by AlexGiroud** [comment]
> Hello, I have the same error than @jminkhar with french language without the internationalization. Do you have a solution ?

A quick solution is to enable internationalization

**Comment by qmonmert** [comment]
@DanielFran fix by #23008 we can close it
